main.tf

```diff
@@ -18,22 +18,23 @@ locals {
   }
 
   criticality = {
-    sbox     = "Low"
-    aat      = "High"
-    stg      = "High"
-    prod     = "High"
-    ithc     = "Medium"
-    test     = "Medium"
-    perftest = "Medium"
-    demo     = "Medium"
-    dev      = "Low"
-    ptl      = "High"
-    preview  = "Low"
-    ldata    = "High"
-    sandbox  = "Low"
-    nle      = "High"
-    nonprod  = "Medium"
-    ptlsbox  = "Low"
+    sbox      = "Low"
+    aat       = "High"
+    stg       = "High"
+    prod      = "High"
+    ithc      = "Medium"
+    test      = "Medium"
+    perftest  = "Medium"
+    demo      = "Medium"
+    dev       = "Low"
+    ptl       = "High"
+    preview   = "Low"
+    ldata     = "High"
+    sandbox   = "Low"
+    nle       = "High"
+    nonprod   = "Medium"
+    ptlsbox   = "Low"
+    sbox-init = "Low"
```

Comment:
I don’t think init is right

Reply:
yeah, but didnt seem to have a choice as the env name is `sbox-init` had a look through some documentation as it's not a common sytax but thought to give it a go.

Reply:
```suggestion
    sbox-int = "Low"
```

Reply:
Thanks @timja did notice the typo. Updated.